### ISSUE TITLE
[FEATURE] Make tournaments be stopped unconditionally

### ISSUE THREAD

**Issue by JoshMatthew** [heroiclabs/nakama#555]
Hello this is a feature request! It would be nice if somehow, tournament creators can stop a tournament unconditionally. Or maybe just the programmers because it's really annoying to wait for the tournament to stop to see if the notifications work. Or maybe this feature will be useful in some other ways.

**Comment by novabyte** [comment]
@JoshMatthew Thanks for the feature suggestion. I'm curious about the specifics of the feature request. Do you need to be able to programmatically complete a tournament early or do you just want to be able to just trigger the tournament end callback when you test the game?

**Comment by JoshMatthew** [comment]
> @JoshMatthew Thanks for the feature suggestion. I'm curious about the specifics of the feature request. Do you need to be able to programmatically complete a tournament early or do you just want to be able to just trigger the tournament end callback when you test the game?

Yeah maybe an api? That when called will trigger the end tournament callback? So just also in development, it's gonna be easier to develop.

**Comment by mofirouz** [comment]
Maybe a button in the Nakama Console on the leaderboard/tournament that emulates the behaviour?

**Comment by JoshMatthew** [comment]
> Maybe a button in the Nakama Console on the leaderboard/tournament that emulates the behaviour?

Yeah that would be convenient

**Comment by novabyte** [comment]
@mofirouz I think it would also be great (though is off topic to this request) to have an entry in the configuration view that shows the current coordinator node for leaderboard/tournament reset hooks.
